tests/src/test/java/org/eclipse/hono/tests/http/HttpTestBase.java

```diff
@@ -163,7 +163,6 @@ public static void init(final TestContext ctx) {
      */
     @Before
     public void setUp() {
-
```

Comment:
can you please remove this change from the commit?

Reply:
This file is no more included in commits.